js/misc/paygManager.js

```diff
@@ -211,6 +213,10 @@ var PaygManager = GObject.registerClass({
         this.emit('code-expired');
     }
 
+    _onImpendingShutdown(proxy, sender, [seconds_remaining, shutdown_reason]) {
+        //TODO notify the user https://phabricator.endlessm.com/T27134
```

Comment:
nit: missing space before `TODO`.

Reply:
I was also gonna say use camelCase for the function arguments, but that's also an inconsequential nitpick

Reply:
Addressed both comments